.github/workflows/cd_push.yml

```diff
@@ -65,7 +65,16 @@ jobs:
           EUC1STAGING_TOKEN: ${{ secrets.ARGOCD_EUC1STAGING_API_AUTH }}
           EUC1_TOKEN: ${{ secrets.ARGOCD_EUC1_API_AUTH }}
 
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.CI_APP_ID }}
```

Comment:
Can we rename this to be 
```
WORKFLOWS_GITHUB_APP_ID
```
And match the private key name to that too? 


It's getting hard to tell which github secret is used where

Reply:
sure